server/pension/translation.py

```diff
@@ -1,11 +1,5 @@
 from modeltranslation.translator import register, TranslationOptions
-from .models import ManagingBody, Fund, Instrument, Issuer
-
-
-@register(ManagingBody)
```

Comment:
@ranmoshe Why you just delete the `ManagingBody`? Shouldn't we need to replace it with `FundManagingBody`?